src/Middleware/Payload.php

```diff
@@ -28,6 +33,15 @@ public function __construct(array $options = [])
     }
 
     /**
+     * If the Request object already has a parsedBody, normally Payload will skip parsing. This is not always
+     * desirable behavior. Calling this setter allows you to override this behavior.
+     */
+    public function overrideExistingParsedBody()
+    {
+        $this->overrideExistingParsedBody = true;
+    }
```

Comment:
Hi!
I'm waiting for this pull request to tag a new version. Are you still interested in working on this?